UI/src/view/link_property/linkPropertyForm.js

```diff
@@ -236,6 +257,24 @@
           '<li><button id="work-list-link-assts-on-expired-links" class="assets-on-expired-links-work-list btn btn-tertiary" onclick=location.href="#work-list/assetsOnExpiredLinks">Poistuneilla tielinkeillä olevat kohteet</button></li>');
     };
 
+    var trafficDirectionChangePopUpConditional = function(originalValue, selectedValue) {
+      return (
+          (originalValue === "BothDirections" && (selectedValue === "AgainstDigitizing" || selectedValue === "TowardsDigitizing")) ||
```

Comment:
enumration löytyis kyl myös this.trafficDirections joten sen stringValue kenttää vois myös hyödyntää tässä.

Reply:
Joo otan käyttöön